tests/export_test.py

```diff
@@ -396,23 +396,39 @@ def test_lowering_parameters_for_export(self):
     # Test that we propagate properly the LoweringParameters.for_export
     test_primitive = core.Primitive("_test_primitive_for_export")
     test_primitive.def_abstract_eval(lambda in_aval: in_aval)
+    # Store here the context for lowering
+    context = {}
     def test_primitive_lowering(ctx, arg):
-      if ctx.module_context.lowering_parameters.for_export:
-        raise ValueError("Lowering for export not supported")
+      nonlocal context
```

Comment:
This `nonlocal` is not necessary, since you are never assigning to `context`, only adding new entries in the dict.